build/compile/Microsoft.DotNet.Cli.LzmaArchive.targets

```diff
@@ -0,0 +1,40 @@
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <Target Name="GenerateNuGetPackagesArchive"
+            DependsOnTargets="SetupNuGetPackagesArchiveInputsOutputs"
+            Inputs="$(Stage2Directory)/sdk/$(SdkVersion)/dotnet.dll"
```

Comment:
I guess `$(RepoRoot)/tools/Archiver` is the other "input" to this target.

---
In reply to: [69323834](https://github.com/dotnet/cli/pull/3737#discussion_r69323834) [](ancestors = 69323834)